src/Visitor/WrappedFunctionInfoCollectorVisitor.php

```diff
@@ -36,6 +36,8 @@ public function enterNode(Node $node)
         if ($this->isFunctionLikeNode($node)) {
             $this->scopeStack[] = $node;
             $node->setAttribute(self::IS_ON_FUNCTION_SIGNATURE, true);
+        }elseif ($this->isPartOfFunctionSignature($node)) {
```

Comment:
Seems like all items that "are part of function signature", e.g. return value, will never get `FUNCTION_SCOPE_KEY` attribute (the next `elseif` section. This is a BC